interim-21-06/README.md

```diff
@@ -8,4 +8,5 @@
 - [cache-trailer.pdf](cache-trailer.pdf)
 - [search.pdf](search.pdf)
 - [signatures.pdf](signatures.pdf)
-- [targeted-cc.pdf](targeted-cc.pdf)
\ No newline at end of file
+- [targeted-cc.pdf](targeted-cc.pdf)
+- [client-cert-field.pdf](client-cert-field.pdf`)
```

Comment:
Just so you know - you don't need to edit README, that's done automagically. No harm tho.